src/styles.scss

```diff
@@ -126,6 +126,12 @@
     background-color: var(--primaryColor) !important
 }
 
+.btn-primary-blue {
+    background: #e3edf1 !important;
```

Comment:
@suvarnakale dont put hardcoded color code for button

Reply:
okay